futures-async-runtime/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "futures-async-runtime"
+version = "0.2.0-beta"
+authors = ["Alex Crichton <alex@alexcrichton.com>"]
+license = "MIT/Apache-2.0"
+readme = "README.md"
+keywords = ["async"]
+categories = ["asynchronous", "rust-patterns"]
+repository = "https://github.com/rust-lang-nursery/futures-rs"
+homepage = "https://github.com/rust-lang-nursery/futures-rs"
+documentation = "https://docs.rs/futures-rs"
+description = """
+Runtime for the async/await macros in the `futures` crate.
+"""
+
+[dependencies.futures-core]
+version = "0.2.0-beta"
+path = "../futures-core"
+
+[dependencies.futures-stable]
+version = "0.2.0-beta"
+path = "../futures-stable"
```

Comment:
These two dependencies should have `default-features = false` to avoid `futures-core/std` and `futures-stable/std` (which then I think recursively activates most of the `std` features except for `futures` itself) always being active when a library depends on `futures = { version = "0.2.0", default-features = false }`.

Reply:
Could you fix this in a rebase of #900? If not, I'll open a second PR. I'm going to merge this now so that we don't have to block fixing other things in other repos on this PR.

Reply:
Sure